### ISSUE TITLE
⛔ ♿ 🛟 Support Request: Tab index is being flagged as higher than 0 when the Ally - Web Accessibility plugin is active

### ISSUE THREAD

**Issue by gx1-hub** [elementor/elementor#30813]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Lighthouse is compelling about the tab index being higher then 0 in some elements. This is an accessibility issue. The Tabindex of the skip-to-content button and the Ally-widget are set to 1. Successfully changed tab index of skip-to-content button to 0 with a script.

But no success with changing the tab index of the Ally-widget, whatever i try.

However, setting tab index of skiptocontent link results in inability to jump from the skiptocontent lbutton to next item like menu with the tab key. It leads to content.

### Steps to reproduce

I added a script to the child functions.php:
```php
add_action('wp_footer', function() {
    echo '<script>
        document.querySelector("a.ea11y-skip-to-content-link")?.setAttribute("tabindex", "0");
    </script>';
});  
```

that changed the tab index of the skip-to content button to 0.

I try to add various codes to try to change the tab index of the Ally-widget to 0, without success, the last was this one:

```php
add_action('wp_footer', function() {
    echo '<script>
        function fixAllyButtonTabIndex() {
            const allyButton = document.querySelector(".ea11y-widget-button--open-widget");
            if (allyButton && allyButton.getAttribute("tabindex") === "1") {
                allyButton.setAttribute("tabindex", "0");
            }
        }
        fixAllyButtonTabIndex();
        const interval = setInterval(fixAllyButtonTabIndex, 500);
        setTimeout(() => clearInterval(interval), 10000);
    </script>';
});
add_action('wp_footer', function() {
    echo '<script>
   document.querySelector(".ea11y-widget-button--open-widget")?.setAttribute("tabindex", "0");
    </script>';
});
```

It looks like the widget is overwriting the tabindex.

### Expected behavior

Expect the tab index to be zero by default, or at least possibility to change it via code and using tab key should link to next tab item instead of content.

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [x] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt


== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Debian 12 v10.11.6-MariaDB-0+deb12u1
	PHP Version: 8.2.28
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.7.2
	Site URL: https://kk-talents.de
	Home URL: https://kk-talents.de
	WP Multisite: No
	Max Upload Size: 2 MB
	Memory limit: 256M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: de_DE
	Timezone: Europe/Berlin
	
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 2.0.0
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.3.0
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: de
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.3.1 Safari/605.1.15

== Active Plugins ==

	Ally - Web Accessibility &amp; Usability
		Version: 3.1.0
		Author: Elementor.com

	Code Snippets
		Version: 3.6.8
		Author: Code Snippets Pro

	Elementor
		Version: 3.28.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.28.2
		Author: Elementor.com

	Temporary Login
		Version: 1.3.0
		Author: Elementor.com

	WP Meteor
		Version: 3.4.15
		Author: Aleksandr Guidrevitch

	WPML Media Translation
		Version: 2.7.7
		Author: OnTheGoSystems

	WPML Multilingual CMS
		Version: 4.7.3
		Author: OnTheGoSystems

	WPML SEO
		Version: 2.1.1
		Author: OnTheGoSystems

	WPML String Translation
		Version: 3.3.2
		Author: OnTheGoSystems


== Eigenschaften ==
	Custom Fonts: 2
	Custom Icons: 0

== Integrations ==
	


== Elementor-Experimente ==
	Inline-Schriftarten-Symbole: Aktiv
	Additional Custom Breakpoints: Inaktiv
	Container: Aktiv
	Optimized Markup: Standardmäßig deaktiviert
	Google Fonts lokal laden: Aktiv
	Default to New Theme Builder: Inaktiv
	Header und Footer: Inaktiv
	Nested Elements: Aktiv
	Editor Top Bar: Aktiv
	Seiten Panel: Standardmäßig deaktiviert
	Element Caching: Inaktiv
	Elementor Home Screen: Standardmäßig aktiviert
	Elementor Editor Events: Standardmäßig deaktiviert
	Atomic Widgets: Standardmäßig deaktiviert
	Globale Klassen: Standardmäßig deaktiviert
	Launchpad Checklist: Standardmäßig aktiviert
	Cloud Library: Standardmäßig deaktiviert
	Menü: Inaktiv


== Protokoll ==
	
Log: showing 20 of 292025-03-24 11:43:28 [info] elementor-pro::elementor_pro_updater Started 
2025-03-24 11:43:28 [info] Elementor Pro/Upgrades - _on_each_version Start  
2025-03-24 11:43:28 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2025-03-24 11:43:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.28.0',
  'to' => '3.28.1',
)]
2025-03-24 11:43:28 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.28.0',
  'to' => '3.28.1',
)]
2025-03-30 18:17:31 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.28.1',
  'to' => '3.28.2',
)]
2025-03-30 18:17:54 [info] elementor::elementor_updater Started 
2025-03-30 18:17:54 [info] Elementor/Upgrades - _on_each_version Start  
2025-03-30 18:17:54 [info] Elementor/Upgrades - _on_each_version Finished 
2025-03-30 18:17:54 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.28.1',
  'to' => '3.28.2',
)]
2025-03-31 06:16:59 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.28.1',
  'to' => '3.28.2',
)]
2025-03-31 06:17:31 [info] elementor-pro::elementor_pro_updater Started 
2025-03-31 06:17:31 [info] Elementor Pro/Upgrades - _on_each_version Start  
2025-03-31 06:17:31 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2025-03-31 06:17:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.28.1',
  'to' => '3.28.2',
)]
2025-04-01 17:05:56 [info] elementor::elementor_updater Started 
2025-04-01 17:05:56 [info] Elementor/Upgrades - _on_each_version Start  
2025-04-01 17:05:56 [info] Elementor/Upgrades - _on_each_version Finished 
2025-04-01 17:05:56 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.28.2',
  'to' => '3.28.3',
)]
2025-04-01 17:05:56 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.28.2',
  'to' => '3.28.3',
)]

JS: showing 5 of 5JS: 2025-03-20 08:02:15 [error X 25][../wp-admin/load-scripts.php?c=0&amp;loadchunk_0=jquery-core,jquery-migrate,wp-hooks&amp;ver=6.7.2:2:28767] undefined is not an object (evaluating &#039;c[0].value&#039;) 
JS: 2025-03-25 11:26:07 [error X 2][../elementor/assets/lib/swiper/v8/swiper.min.js?ver=8.4.5:13:4342] undefined is not a function (near &#039;...[a,i,r,n]...&#039;) 
JS: 2025-03-26 14:12:58 [error X 1][../elementor/assets/js/editor.min.js?ver=3.28.1:3:607598] null is not an object (evaluating &#039;elementor.documents.getCurrent().id&#039;) 
JS: 2025-03-26 19:15:56 [error X 9][../elementor-pro/assets/js/editor.min.js:2:68381] Object.entries requires that input parameter not be null or undefined 
JS: 2025-03-26 22:00:47 [error X 3][../elementor/assets/js/common-modules.min.js?ver=3.28.1:2:64007] Can&#039;t find variable: elementor 

PHP: showing 9 of 9PHP: 2025-03-26 06:39:08 [warning X 1][../elementor/core/files/file-types/svg.php::161] simplexml_load_file(): I/O warning : failed to load external entity &quot;../wp-content/uploads/Her-Schmitt.svg&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-03-26 07:03:11 [warning X 2][../elementor/includes/managers/image.php::113] Trying to access array offset on false [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-03-26 10:48:31 [warning X 1][../elementor/core/files/file-types/svg.php::161] simplexml_load_file(): I/O warning : failed to load external entity &quot;../wp-content/uploads/Herr-Schmitt.svg&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-03-26 10:56:36 [warning X 1][../elementor/core/files/file-types/svg.php::161] simplexml_load_file(): I/O warning : failed to load external entity &quot;../wp-content/uploads/Timeline-Deutsch.svg&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-03-26 11:11:39 [warning X 1][../elementor/core/files/file-types/svg.php::161] simplexml_load_file(): I/O warning : failed to load external entity &quot;../wp-content/uploads/Timeline-Deutsch-1.svg&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-03-26 15:42:08 [warning X 1][../elementor/core/files/file-types/svg.php::161] simplexml_load_file(): I/O warning : failed to load external entity &quot;../wp-content/uploads/Timeline-Deutsch-2.svg&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-03-26 15:58:25 [warning X 3][../elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: ../elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../elementor/data/base/endpoint.php(156): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2025-03-26 15:58:26 [notice X 1][../elementor-pro/modules/screenshots/module.php::33] Function utf8_decode() is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-03-28 09:06:06 [warning X 3][../elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Kompatibel

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [x] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @gx1-hub!

Thank you for reaching out! 🙏 

Please, keep in mind that this repository is reserved for handling [Elementor Editor Bug Reports](https://elemn.to/gh-contributing-bug-reports) only, and is not meant to be used for reporting issues with other plugins, including the [Ally - Web Accessibility](https://wordpress.org/plugins/pojo-accessibility/) plugin.

For issues with this plugin, please use one of the following communication channels:

1. [WordPress Directory Support Forum](https://wordpress.org/support/plugin/pojo-accessibility/) (for Fee users)
2. [Elementor Support](https://my.elementor.com/care/support) (for Paying customers)

If you have a [paid subscription](https://elementor.com/products/ally-web-accessibility/) of the Ally plugin, please [reach out to Elementor Support](https://elemn.to/support-ticket). Otherwise, submit your [Support Request in the WordPress Directory](https://wordpress.org/support/plugin/pojo-accessibility/). Make sure to search for existing posts before submitting your own to ensure a quicker resolution. 

I will, however, share this issue with the Ally team only this time as an exception. Future issues regarding any other plugin that isn't the [Elementor Website Builder](https://wordpress.org/plugins/elementor/) will be closed without any action as defined in our [Contribution Guidelines](https://elemn.to/contributing).

### 👍 For now, that's all we can help with. 

Kind regards
